### DISCUSSION TITLE
Loading SQLite scripts overwrites tab content.

### DISCUSSION THREAD

**Discussion by DJCodex** [pawelsalawa/sqlitestudio#4860] [Q&A]
Existe uma pequena falha, que acredito ser simples de resolver. Acabei de pegar o SOURCE e estou tentando entender o código. Porém a falha é a seguinte:
Quando usamos um CTRL+O para carregar um script SQL de um arquivo. O conteúdo do arquivo é carregado perfeitamente. Porém se tivermos uma aba aberta, com um outro arquivo nela. O conteúdo da será sobreposto pelo novo arquivo que está sendo carregado. O que torna o uso um pouco quanto confuso.
Acredito que a solução seja simples. Já bastaria abrir uma nova aba, e carregar o arquivo nesta aba recém aberta. Porém como acabei de ter acesso ao SOURCE ainda não encontrei o ponto correto para fazer a mudança. De qualquer forma fica registrado, para possível correção.

**Comment by pawelsalawa** [comment]
Translated to English:
> There is a small flaw, which I believe is simple to resolve. I just got SOURCE and I'm trying to understand the code. But the error is as follows:
> When we use a CTRL+O to load an SQL script from a file. The file content is loaded perfectly. But if we have a tab open, with another file in it. The contents of the will be overwritten by the new file being loaded. Which makes using it a bit confusing.
> I believe the solution is simple. It would be enough to open a new tab, and load the file in this newly opened tab. However, as I just had access to SOURCE, I still haven't found the correct point to make the change. In any case, it is registered, for possible correction.

**Reply by pawelsalawa** [reply]
I've created #4861 to track and fix it.
